### DISCUSSION TITLE
Reprogramming N64 CIC on repros

### DISCUSSION THREAD

**Discussion by JCreazy** [sanni/cartreader#396] [Need help]
Hello everyone.  I am not sure how many people have attempted this but I am attempting to reflash the CIC on one of my repros using the instructions [here.](https://github.com/sanni/cartreader/wiki/Reflashing-a-N64-Repro)  I ended up using platform.io to upload the sketch the Arduino Uno because the Arduino software was having trouble compiling. Anyway, I have connected  all the dupont connectors to the correct locations.  I run the .bat file that I created (I changed the port to the correct one) I just get the error of 

``avrdude.exe: stk500_recv(): programmer is not responding
avrdude.exe: stk500_getsync() attempt 1 of 10: not in sync: resp=0x38`` 

It goes through all the attempts and then just says it's done.  

Any advice?

Also, out of curiosities sake. Is it at all possible to incorporate this into the cart reader? I know uses a different Arduino but I think most of the pins of the CIC chip can be accessed through the cartridge slot.

Thanks

**Comment by sanni** [comment]
The CIC pins are not connected on the Cart Reader PCB, but you could wire them up manually and try it. It could work.

**Comment by KobeOne2** [comment]
@JCreazy Did you find a solution for this? I'm having the same problem.

**Comment by KobeOne2** [comment]
Found the mistake. It is required with most Arduino boards (UNO, Mega, ...) to put a 10µF electrolytic capacitor between RESET and GND with the positive (long leg in most cases) connected to RESET.

Btw: Is there a tool to check which save type a (e.g. ROM hack) N64 ROM uses?

Edit: Sorry, I seem to have updated the wiki article for flashing N64 repros; I figured I couldn't edit the live version and could only make a pull request or something.

**Reply by sanni** [reply]
Thanks for adding this info to the wiki. 👍
